ci/Jenkinsfile

```diff
@@ -164,9 +164,13 @@ pipeline {
       } } }
     }
   }
+
   post {
     always { script {
       archiveArtifacts('aut/*.log')
+      addAllureCategories()
+      addEnvironmentSection()
+
```

Comment:
Why are we even doing this here? Shouldn't this be a step done by the Python code if this is supposed to be part of results?

Reply:
And if it has to be here, why make two functions that make a single `sh` call? Why not just add it in the `always` block with a comment explaining what it does?

Functions make sense when they are reused or hide complexity that doesn't need to be seen, this does neither. Might make more sense if it was one function and not two with a name that explains the purpose.

Also, this might as well be done in the `Prep` stage.

Reply:
Ok. I added it in the always block with a comment